src/views/MyRequests/Table.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { useQuery } from '@apollo/client';
+import { Link as UswdsLink } from '@trussworks/react-uswds';
+import GetRequestsQuery from 'queries/GetRequestsQuery';
+import { GetRequests } from 'queries/types/GetRequests';
+
+const Table = () => {
+  const { loading, error, data } = useQuery<GetRequests>(GetRequestsQuery, {
```

Comment:
nitpick, useQuery can take two type args, `useQuery<GetRequests, GetRequestsVariables>` which helps with type checking the input variables